src/wombats/daos/game.clj

```diff
@@ -29,6 +29,13 @@
 ;; ↓   ↑
 ;; 2 → ↑
 
+(def game-projection
+  '[*
```

Comment:
Wildcard projection in Datomic. It's quoted because it's not valid Clojure so it shouldn't be evaluated.